Userland/Applications/TodoList/TaskListModel.cpp

```diff
@@ -0,0 +1,72 @@
+/*
+ * Copyright (c) 2021, Arthur Brainville <ybalrid@ybalrid.info>
+ *
+ * SPDX-License-Identifier: BSD-2-Clause
+ */
+
+#include "TaskListModel.h"
+
+#include <AK/JsonArray.h>
+#include <AK/JsonObject.h>
+#include <AK/JsonParser.h>
+#include <AK/JsonValue.h>
+
+#include <cstdio>
+
+void TaskListModel::load_from_disk(AK::String path)
+{
+    //TODO There's probably a better API in Serenity than the stdio for that
+    FILE* todofile = fopen(path.characters(), "r");
+    StringBuilder jsonBuilder;
+    if (todofile) {
+        for (;;) {
+            char buffer[1024];
+            if (!fgets(buffer, sizeof(buffer), todofile))
+                break;
+            jsonBuilder.append(buffer);
+        }
+
+        AK::JsonValue todoJson = AK::JsonValue::from_string(jsonBuilder.to_string()).value();
+
+        if (todoJson.is_array()) {
+            auto list = todoJson.as_array();
+            for (size_t i = 0; i < list.size(); ++i) {
+                const auto& task = list[i].as_object();
+                String title = task.get("title").to_string();
+                String description = task.get("description").to_string();
+                const Task::State state = task.get("state").to_string() == "DONE" ? Task::State::done : Task::State::todo;
+
+                Task new_task_object(title, description);
+                new_task_object.set_state(state);
+                add_task(new_task_object);
+            }
+        }
+    }
+}
+
+void TaskListModel::save_to_disk(AK::String path) const
+{
+    AK::JsonArray json_task_array;
+    printf("Task list size : %ld\n", task_list.size());
```

Comment:
```suggestion
    dbgln("Task list size: {}", task_list.size());
```

Please use `dbgln` instead of `printf`.